### ISSUE TITLE
[display] incorrect EnumFieldOrigin on hover

### ISSUE THREAD

**Issue by Gama11** [HaxeFoundation/haxe#7164]
The origin of statically imported enum fields in completion is correct:

![](https://i.imgur.com/7sFkAyb.png)

However, in hover, it's not (it's missing "statically imported" / origin is 0 instead of 1):

![](https://i.imgur.com/cUfn47b.png)

**Comment by Simn** [comment]
This doesn't work for class fields either, right?

**Comment by Gama11** [comment]
Oh, you're right: it only works in completion there too.

**Comment by Simn** [comment]
Yeah, this is a bit annoying. We're gonna have to play "where did it come from" for cases like this. It's fine for completion because that looks at all potential sources anyway, whereas here we already have our target.

**Comment by Simn** [comment]
Moving back to 4.1 because it's quite annoying, but not a big deal for the user.
